apps/calendar/test/marionette/server_test.js

```diff
@@ -9,6 +9,9 @@ var Calendar = require('./lib/calendar'),
 var calendarName = 'firefox-os';
 
 marionette('interop basic', function() {
+  /* jshint -W027*/
```

Comment:
This was broken before and was only being run locally. Would like to resurrect, but it's just slowing me down right now